types/handler.go

```diff
@@ -11,7 +9,7 @@ type AnteHandler func(ctx Context, tx Tx, simulate bool) (newCtx Context, err er
 
 // PostHandler like AnteHandler but it executes after RunMsgs. Runs on success
 // or failure and enables use cases like gas refunding.
-type PostHandler func(ctx Context, tx Tx, msgResponses []*codectypes.Any, simulate, success bool) (newCtx Context, err error)
+type PostHandler func(ctx Context, tx Tx, gasMeter GasMeter, simulate, success bool) (newCtx Context, err error)
```

Comment:
No need since it's already available from the `ctx`

```suggestion
type PostHandler func(ctx Context, tx Tx, simulate, success bool) (newCtx Context, err error)
```